examples/torch/classification/configs/sparsity_quantization/resnet50_imagenet_rb_sparsity50_int8.json

```diff
@@ -5,7 +5,7 @@
         "sample_size": [1, 3, 224, 224]
     },
     "num_classes": 1000,
-    "batch_size": 640, // 64 per GPU
+    "batch_size": 256,
```

Comment:
This should be tested by weekly training test, I guess

Reply:
yes, but it's strange value for training (10 GPU according to the comment) and too much for eval. I just aligned with similar config - https://github.com/openvinotoolkit/nncf/blob/develop/examples/torch/classification/configs/sparsity_quantization/resnet50_imagenet_rb_sparsity_int8.json#L8

Reply:
Ok, I just wanted to highlight we should validate it at some point

Reply:
anyway, if we had a weekly test for this, we would haven't been able to train with such batch